docs/USAGE-GUIDE.md

```diff
@@ -603,6 +603,55 @@ a pointer to abitrary data for use within the callback, a pointer to a key
 which can be used to delete the cached entry, and a 64 bit unsigned integer
 specifying the size of this key.
 
+
+## Record sizes
+
+### Throughput vs Latency
+
+When sending data, s2n-tls uses a default maximum record size which experimentation
+has suggested provides a reasonable balance of performance and throughput.
+
+**s2n_connection_prefer_throughput** can be called to increase the record size, which
+minimizes overhead. It also increases s2n-tls's memory usage.
+
+**s2n_connection_prefer_low_latency** can be called to decrease the record size, which
+allows the receiver to decrypt the data faster. It also decreases s2n-tls's memory usage.
+
+These options only affect the size of the records that s2n-tls sends, not the behavior
+of the peer.
+
+### Maximum Fragment Length
+
+The maximum number of bytes that can be sent in a TLS record is called the "maximum fragment length",
+and is set to 2^14 bytes by default. Regardless of the maximum record size that s2n-tls
+uses when sending, it may receive records containing up to 2^14 bytes of plaintext.
+
+A client can request a lower maximum fragment length by calling **s2n_config_send_max_fragment_length**,
```

Comment:
Can you be more explicit about whether this API affects the records sent or the records received?